lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -94,15 +97,22 @@ class ReportRenderer {
     element.querySelector('.lh-score__description')
         .appendChild(this._dom.createSpanFromMarkdown(description));
 
-    return element;
+    return /** @type {!Element} **/ (element);
+  }
+
+  /**
+   * @param {!Document|!Element} context
```

Comment:
maybe a function description for why you'd want to inject this? there won't be any hints on the LH side